test/unit/carriers/shipwire_test.rb

```diff
@@ -24,67 +24,65 @@ def test_response_with_no_rates_is_unsuccessful
 
   def test_successfully_get_international_rates
     date = DateTime.new(2011, 8, 1)
-    DateTime.expects(:now).returns(date).at_least_once
     @carrier.expects(:ssl_post).returns(xml_fixture('shipwire/international_rates_response'))
 
-    response = @carrier.find_rates(
-                 location_fixtures[:ottawa],
-                 location_fixtures[:london],
-                 package_fixtures.values_at(:book, :wii),
-                 :order_id => '#1000',
-                 :items => @items
-               )
-
-    assert response.success?
-
-    assert_equal 1, response.rates.size
-
-    assert international = response.rates.first
-    assert_equal "INTL", international.service_code
-    assert_equal "UPS", international.carrier
-    assert_equal "UPS Standard", international.service_name
-    assert_equal 2806, international.total_price
-    assert_equal date + 7, international.delivery_date
-    assert_equal [date + 1, date + 7], international.delivery_range
+    Timecop.freeze(date) do
+      response = @carrier.find_rates(
+                   location_fixtures[:ottawa],
+                   location_fixtures[:london],
+                   package_fixtures.values_at(:book, :wii),
+                   :order_id => '#1000',
+                   :items => @items
+                 )
+
+      assert response.success?
+
+      assert_equal 1, response.rates.size
+
+      assert international = response.rates.first
+      assert_equal "INTL", international.service_code
+      assert_equal "UPS", international.carrier
+      assert_equal "UPS Standard", international.service_name
+      assert_equal 2806, international.total_price
+      assert_equal [date + 1.day, date + 7.days], international.delivery_range
```

Comment:
Behaviour depends on the class.

```
irb(main):011:0> time = Time.now
=> 2016-07-04 14:38:02 -0400
irb(main):012:0> datetime = time.to_datetime
=> #<DateTime: 2016-07-04T14:38:02-04:00 ((2457574j,67082s,766271000n),-14400s,2299161j)>
irb(main):013:0> time + 1
=> 2016-07-04 14:38:03 -0400
irb(main):014:0> datetime + 1
=> #<DateTime: 2016-07-05T14:38:02-04:00 ((2457575j,67082s,766271000n),-14400s,2299161j)>
```

One of our Rails 5 bugs is around returning `DateTime` vs `Time` when doing time math.